sdk/storage/azblob/appendblob/client.go

```diff
@@ -65,12 +68,13 @@ func NewClientWithSharedKeyCredential(blobURL string, cred *blob.SharedKeyCreden
 	authPolicy := exported.NewSharedKeyCredPolicy(cred)
 	conOptions := shared.GetClientOptions(options)
 	conOptions.PerRetryPolicies = append(conOptions.PerRetryPolicies, authPolicy)
```

Comment:
The `authPolicy` should go in the `runtime.PipelineOptions.PerRetryPolicies` field (this was missed in earlier reviews).

Reply:
Fixed!